lib/livekit_client.dart

```diff
@@ -20,3 +18,5 @@ export 'src/track/track_publication.dart';
 export 'src/track/local_track_publication.dart';
 export 'src/track/remote_track_publication.dart';
 export 'src/widget/video_track_renderer.dart';
+export 'src/proto/livekit_rtc.pb.dart';
```

Comment:
I don't know if we want to export this entire file. likely similar the below. The protobufs are more like internal implementation details.

Reply:
I thought so too but `VideoQuality` is referenced from the example app and I cant remove that line, so I might just **show** `VideoQuality` for now.

Reply:
for now `VideoQuality` and `TrackType` are exposed.